cupertino_gallery/lib/widgets/switch_page.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/cupertino.dart';
+
+class SwitchPage extends StatefulWidget {
+  const SwitchPage({super.key});
+
+  @override
+  State<SwitchPage> createState() => _SwitchPageState();
+}
+
+class _SwitchPageState extends State<SwitchPage> {
+  bool _value = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text('Switch'),
+      ),
+      child: Center(
+        child: CupertinoSwitch(
```

Comment:
Since we are showing CupertinoSwitch, showing CupertinoRadio and Checkbox might be a good idea too.